javascript/javascript1/week3/note.js

```diff
@@ -0,0 +1,83 @@
+// insert your code here:
+//todo :forth exercise
+
+//Smart-ease - Back to the basics!
+
+//?Create a variable called notes
+let notes = [];
+
+//!OBJECT CONSTRUCTURE
+// const SaveNote = function (content, id) {
+//   (this.content = content), (this.id = id);
+// };
+// const note1 = new SaveNote("Pick up groceries", 1);
+// const note2 = new SaveNote("Do laundry", 2);
+
+// notes.push(note1);
+// notes.push(note2);
+// console.log(notes.note1);
+
+//?Create a function called saveNote
+const myObj = {};
+function saveNote(content, id) {
+  if (content) {
+    const snote = {
+      content: content,
+      id: id,
+    };
+    notes.push(snote);
+  }
+}
+saveNote("Pick up groceries", 1);
+saveNote("Do laundry", 2);
+
+console.log("notes= ", notes);
+//? Get a note
+function getNote(id) {
+  let specialObj = {};
+  notes.forEach((el) => {
+    if (el.id === id) {
+      specialObj = el;
```

Comment:
Here you can return from forEach when you find the element, so it doesn't spend time looping through the rest of the elements.